tools/signing.py

```diff
@@ -22,13 +22,13 @@ def main():
     if args.mode == "header":
         val = ""
         try:
-            with open(args.publickey, "r") as f:
-                pub = f.read()
+            with open(args.publickey, "rb") as f:
+                pub = b"\n".join(map(lambda x: x.rstrip(), f.readlines()))
```

Comment:
This changes nothing in regards to my objection of this patch.
If anything, this makes it worse, as this replaces any sequence of `r'[\r\t ]*\n'` with `'\n'`.

As I've understood this, the key may be in either binary or ascii armor format.

```suggestion
                pub = f.read()
```